Cargo.toml

```diff
@@ -21,8 +21,16 @@ categories = ["network-programming", "data-structures"]
 
 [dependencies]
 byteorder = "1.0.0"
-iovec = "0.1"
 serde = { version = "1.0", optional = true }
 
+[dependencies.iovec]
+version = "0.1"
+optional = true
+
 [dev-dependencies]
 serde_test = "1.0"
+
+[features]
+default = ["use_std"]
+use_alloc = []
+use_std = ["iovec"]
```

Comment:
It seems that `use_std` implies `use_alloc`. I'm not sure if there is a way to define this in Cargo.toml.